packages/lib/src/vis/hooks.ts

```diff
@@ -33,7 +33,7 @@ export const useValueToIndexScale = createMemo(getValueToIndexScale);
 export const useAxisDomain = createMemo(getAxisDomain);
 
 export function useDomain(
-  valuesArray: AnyNumArray,
+  valuesArray: AnyNumArray | undefined,
```

Comment:
Subtle change: I need to be able to pass a potentially `undefined` array to `useDomain`. For consistency, I make the same change in `useDomains` and `getDomain`.